src/dpop.js

```diff
@@ -1,6 +1,39 @@
 /* @flow */
 /* eslint-disable promise/no-native, no-restricted-globals */
 
+type KeyPair = {|
+  privateKey: mixed,
+  publicKey: mixed,
+|};
```

Comment:
not yet supported in flow - https://github.com/facebook/flow/blame/aa9112fd181e484ef63db0f7c2593d0c547046e6/lib/bom.js#L40-L47